### DISCUSSION TITLE
Import error/Problem with dll on python

### DISCUSSION THREAD

**Discussion by wuyx05** [DaveSkender/Stock.Indicators#1162] [Help and support]
Hi I am importing the package on python and this is the error I got.
 I am on windows and .net 6.0.418
Any help is appreciated

![image](https://github.com/DaveSkender/Stock.Indicators/assets/57379685/197d4af5-3332-4285-855b-5c517936a5fb)

**Comment by DaveSkender** [comment]
I suspect this is because we don’t host `stock-indicators` in `Conda` package manager.  Try these two commands to see if it’ll load from PyPI

```bash
conda config --set pip_interop_enabled True
pip install stock-indicators
```

I may be way off here,  @LeeDongGeon1996 any suggestions?

**Reply by DaveSkender** [reply]
@wuyx05 were you able to resolve this issue?  If so, please share some insights.  We’re tracking a similar issue with initial setup (https://github.com/DaveSkender/Stock.Indicators.Python/issues/357) and would like to improve our documentation (or code setup) where possible, with your feedback.

**Reply by LeeDongGeon1996** [reply]
Oh, I missed this thread. Might be not a problem of `Conda`, because I'm also using Conda environment when I develop.

**Reply by DaveSkender** [reply]
@wuyx05 here's the exact steps I took on a new Windows 11 laptop recently, to get setup and to run the library:
[QuickStart for Stock Indicators for Python](https://github.com/DaveSkender/Stock.Indicators.Python.Quickstart/#readme)

**Reply by wuyx05** [reply]
@DaveSkender hey i managed to get the same output as in the quick start guide. Thanks a lot
